etl/steps/data/garden/emdat/2022-11-24/shared.py

```diff
@@ -0,0 +1,597 @@
+    countries_set -= set(countries_excluded)
+
+    # Convert set of countries into a sorted list.
+    countries = sorted(countries_set)
+
+    return countries
+
+
+def load_population(regions: Optional[Dict[Any, Any]] = None) -> pd.DataFrame:
+    """Load OWID population dataset, and add historical regions to it.
+
+    Returns
+    -------
+    population : pd.DataFrame
+        Population dataset.
+
+    """
+    # Load population dataset.
+    population = catalog.Dataset(DATA_DIR / "garden/owid/latest/key_indicators/")["population"].reset_index()[
+        ["country", "year", "population"]
+    ]
+
+    # Add data for historical regions (if not in population) by adding the population of its current successors.
+    countries_with_population = population["country"].unique()
+
+    # Consider additional regions (e.g. historical regions).
+    if regions is None:
+        regions = {}
+    missing_countries = [country for country in regions if country not in countries_with_population]
+    for country in missing_countries:
+        members = regions[country]["regions_included"]
+        _population = (
+            population[population["country"].isin(members)]
+            .groupby("year")
+            .agg({"population": "sum", "country": "nunique"})
+            .reset_index()
+        )
+        # Select only years for which we have data for all member countries.
+        _population = _population[_population["country"] == len(members)].reset_index(drop=True)
+        _population["country"] = country
+        population = pd.concat([population, _population], ignore_index=True).reset_index(drop=True)
+
+    error = "Duplicate country-years found in population. Check if historical regions changed."
+    assert population[population.duplicated(subset=["country", "year"])].empty, error
+
+    return cast(pd.DataFrame, population)
+
+
+def load_income_groups() -> pd.DataFrame:
+    """Load dataset of income groups and add historical regions to it.
+
+    Returns
+    -------
+    income_groups : pd.DataFrame
+        Income groups data.
+
+    """
+    income_groups = (
+        catalog.find(
+            table="wb_income_group",
+            dataset="wb_income",
+            namespace="wb",
+            channels=["garden"],
+        )
+        .load()
+        .reset_index()
+    )
+    # Add historical regions to income groups.
+    for historic_region in HISTORIC_TO_CURRENT_REGION:
+        historic_region_income_group = HISTORIC_TO_CURRENT_REGION[historic_region]["income_group"]
+        if historic_region not in income_groups["country"]:
+            historic_region_df = pd.DataFrame(
+                {
+                    "country": [historic_region],
+                    "income_group": [historic_region_income_group],
+                }
+            )
+            income_groups = pd.concat([income_groups, historic_region_df], ignore_index=True)
+
+    return cast(pd.DataFrame, income_groups)
+
+
+def add_population(
```

Comment:
I think it would make sense to add historical regions to our population dataset. That would make everyone's lives easier. And there should be a way to tell whether a region is historical or not (and even better, whether regions belong to other regions).